### PR TITLE
Fix CATSecVertex V3( miniaod SecVtx)

### PR DESCRIPTION
**Pull request by geonmo**
- Fix CATSecVertex V3.
  - Use CATSecVtx directly.

### CODE REVIEW

File: CatAnalyzer/test/catJpsiDraw/PAT2CAT_Modified_cfg.py [hunk 1]
```diff
@@ -82,7 +79,9 @@
 if not options.inputFiles:
     if useMiniAOD:
         if runGenTop:
-            process.source.fileNames = ['/store/relval/CMSSW_7_4_15/RelValTTbar_13/MINIAODSIM/PU25ns_74X_mcRun2_asymptotic_v2-v1/00000/0253820F-4772-E511-ADD3-002618943856.root']
+            process.source.fileNames = [
+                                      '/store/relval/CMSSW_7_4_15/RelValTTbar_13/MINIAODSIM/PU25ns_74X_mcRun2_asymptotic_v2-v1/00000/0253820F-4772-E511-ADD3-002618943856.root',
```
Comment (jshlee) [inline CatAnalyzer/test/catJpsiDraw/PAT2CAT_Modified_cfg.py]: i would change the relval to 76x

**Comment by jshlee** [comment]
i think it would be better to remove PAT2CAT_Modified_cfg.py
and important changes to the main one, with most of the settings in the sec prod cfi

**Comment by geonmo** [comment]
@jshlee Ok, I removed PAT2CAT_Modified_cfg.py. It is just temporary config file.

**Comment by jshlee** [comment]
+1

File: CatProducer/plugins/CATSecVertexProducer.cc [hunk 2]
```diff
@@ -196,21 +196,43 @@ cat::CATSecVertexProducer::produce(edm::Event & iEvent, const edm::EventSetup &
...
-      //const reco::TransientTrack  transTrack1, transTrack2;
-      if ( cand1->get<TrackRef>().isNull() || cand2->get<TrackRef>().isNull()) continue;
-      auto ttrack1 = trackBuilder->build(cand1->get<TrackRef>());
-      auto ttrack2 = trackBuilder->build(cand2->get<TrackRef>());
-      fitTransientTracks(out_, pv, ttrack1, ttrack2, pdgId);
+      else if ( std::abs(cand1->pdgId()) == 13 || std::abs(cand2->pdgId()) == 13) pdgId = 13;
+      else pdgId = 221;
+      //if ( pdgId !=11 && pdgId !=13 ) continue;
+
+      VertexCompositeCandidate aVC = VertexCompositeCandidate( sv->charge(), sv->p4(), sv->vertex(), sv->vertexCovariance(), sv->vertexChi2(), sv->vertexNdof(), pdgId, 0, true);
```
Comment (jhgoh) [inline CatProducer/plugins/CATSecVertexProducer.cc:211]: shouldn't pdgId=443?

Comment (geonmo) [inline CatProducer/plugins/CATSecVertexProducer.cc:211]: I mean that those tracks are not lepton PF candidates.(determine to pion)
Frankly speaking, pdgId=221 is not used at this producer.

**Comment by jshlee** [comment]
do you want both this and #407 merged?

**Comment by geonmo** [comment]
Hello, Jason. Could you merge #407 first?

**Pull request closed without merging**